packages/nimble-components/src/select/testing/select.pageobject.ts

```diff
@@ -53,9 +54,19 @@ export class SelectPageObject {
     }
 
     public getSelectedOption(): ListOption | null {
+        if (this.selectElement.open) {
+            return this.getSelectedDropdownOption();
+        }
```

Comment:
Perhaps I don't understand all the usages of `getSelectedOption`, but it seems misleading to return the active element when it hasn't actually been selected yet. Should this function be left unchanged from master and a new `getActiveOption` function be added to the page object?

Reply:
Done. I was concerned that there were going to be some callers in of this method that may have been relying on the initial behavior, but a) there aren't, and b) that isn't a valid reason to create a disingenuous method.